### PR TITLE
Fix broken CloudinaryDotNet functionality on dotnetfiddle.net

### PR DESCRIPTION
**Pull request by RTLcoil**

### CODE REVIEW

File: Shared/CloudinaryVersion.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+{
+    /// <summary>
+    /// Contains information about release version.
+    /// </summary>
+    internal static class CloudinaryVersion
+    {
+        /// <summary>
+        /// Get full version number of .NET SDK.
+        /// </summary>
+        public static string Full => "1.11.0";
```
Comment (asisayag2) [inline Shared/CloudinaryVersion.cs:11]: What supposed to update this when the publish scripts are running?

Comment (RTLcoil) [reply Shared/CloudinaryVersion.cs:11]: Although from @const-cloudinary 's past comments we understood that manual version change is acceptable, it makes sense to automate this step during publish. We will do it in the current sprint.

Comment (RTLcoil) [reply Shared/CloudinaryVersion.cs:11]: If @const-cloudinary is OK with it as well.

Comment (const-cloudinary) [reply Shared/CloudinaryVersion.cs]: @asisayag2 , @RTLcoil , there is a set_version.ps1 file that updates the version, I updated it.

Comment (const-cloudinary) [reply Shared/CloudinaryVersion.cs]: @RTLcoil and version should be current version, when new version is released, a separate commit will be made.

Comment (RTLcoil) [reply Shared/CloudinaryVersion.cs]: Cool, @const-cloudinary . So you'll let us know or would you like us to ping you on this at some point?

Comment (const-cloudinary) [reply Shared/CloudinaryVersion.cs]: @RTLcoil it all done, nothing to do here

**Pull request merged**